.config.yml

```diff
@@ -3,6 +3,7 @@ excludeOnPrivateRepo:
   - Package-README.md
   - LICENSE.md
   - CONTRIBUTING.md
+  - SECURITY.md
```

Comment:
I'm guessing GitHub only cares about the version that's on the default branch, so shouldn't this item be in the `excludeOnReleaseBranch` collection too?

Reply:
Makes sense to me. I have added it to that list as well.